web/tests/functional/__init__.py

```diff
@@ -22,10 +21,5 @@
 # by the tests.
 PKG_ROOT = os.path.join(REPO_ROOT, 'build', 'CodeChecker')
 
-__LAYOUT_FILE_PATH = os.path.join(PKG_ROOT, 'config', 'package_layout.json')
-with open(__LAYOUT_FILE_PATH) as layout_file:
-    __PACKAGE_LAYOUT = json.load(layout_file)
-sys.path.append(os.path.join(
-    PKG_ROOT, __PACKAGE_LAYOUT['static']['gencodechecker']))
-
+sys.path.append(os.path.join(PKG_ROOT, 'lib', 'python2.7', 'codechecker_api'))
```

Comment:
Can we use the same path for the tests? In one case we use the generated modules from the package in another case we use them from the generated place:
```sys.path.append(os.path.join(BUILD_DIR, "thrift", "v6", "gen-py"))```
and
```sys.path.append(os.path.join(PKG_ROOT, 'lib', 'python2.7', 'codechecker_api'))```